enterprise/migrations/0099_auto_20200702_1537.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.14 on 2020-07-02 15:37
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('enterprise', '0098_auto_20200629_1756'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='enterprisecourseenrollment',
+            name='marked_done',
+            field=models.NullBooleanField(default=False, help_text='Specifies whether a user marked this course as completed in the learner portal.'),
+        ),
+        migrations.AlterField(
```

Comment:
do we need a models.py change for this one too or is it already covered?

Reply:
The `historicalenterprisecourseenrollement` change is automatically created when `enterprisecourseenrollement` changes, so no additional model was changed. :)